### DISCUSSION TITLE
Receive wall messages

### DISCUSSION THREAD

**Discussion by teapot9** [kovidgoyal/kitty#7141] [Q&A] [answered]
Hi,

When I use the `wall` command from `util-linux`, I am not receiving the message on my kitty terminals.
However, the message is received on other terminals such as xterm, kde konsole, ssh sessions (doesn't work on gnome terminal either).

Is there any way to configure kitty to receive such messages?

Thanks

**Comment by kovidgoyal** [comment, marked as answer]
You really shouldn't use wall, it means you have multiple independent
processes writing to the same tty device which can cause undefined
behavior. As for it not working with kitty, it's because kitty does not
modify the utmp database of "logged in" users for every terminal you
create. Since kitty is designed for use with large numbers of
windows/splits/tabs doing so is not a good idea.

**Reply by teapot9** [reply]
The process using `wall` is `nut` to notify when my UPS goes on battery or is low on battery.
I was wondering why I didn't receive the notification on kitty.

Thanks for the explanation!
I'll find a workaround because I kinda need those alerts.

**Comment by kovidgoyal** [comment]
Use notify-send

**Reply by teapot9** [reply]
I can't make `notify-send` work as root because I can't guess where the dbus socket is located (random file `/tmp/dbus-xxxxxxxx`)

**Reply by kovidgoyal** [reply]
Why would you need to guess. Have a login script run under your user account and store the value of the env var in some well known file.

**Reply by teapot9** [reply]
Thanks, I'll try this
